config/routes.rb

```diff
@@ -3,8 +3,9 @@
 
   namespace :api do
     namespace :v1 do
-      delete 'users/games', to: 'users/games#destroy'
```

Comment:
What would happen if we changed all of these to be in the resources users? Not sure if you've tried that or not. Not a big deal either way.

Reply:
I have tried that but it turns the route into `/api/v1/users/:user_id/games` and not `/api/v1/users/games`

Last time I looked at the FE they were using the `/api/v1/users/games` route unless that changed?

Reply:
I told them that we were asking for the ID in the path for these as of this morning. It's Restful I think so I kept it that way.